packages/presenters/ContextWidgetPresenter.js

```diff
@@ -1,8 +1,13 @@
 // TODO: Get "t" from current i18n instance so it doesn't need to be passed
 // import i18n from 'react-i18next'
-import { COMMON_VIEWS } from '@hylo/presenters/CustomViewPresenter'
 
-export default function ContextWidgetPresenter (widget, { t }) {
+// t / translate is currently set to a dummy function if not sent
+const tDummy = t => {
```

Comment:
I'd rather be manually pushing t into presenters and being closer to the correct UI than this approach. 

I haven't push on this much yet, aside from running afoul of the rules of hooks.

Reply:
Absolutely! Yes good catch. This is very definitely a messy in-development bypass which definitely will be removed before live. I should have and still should make that much more clear here with a "TODO!" comment of a sort.